src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -212,7 +212,7 @@ def run_job(self, params):
         return self.test_job_id
 
     def run_job_batch(self, batch_job_inputs, batch_params):
-        child_job_ids = [self.test_job_id for i in range(len(batch_job_inputs))]
+        child_job_ids = [self.test_job_id + f"_child_{i}" for i in range(len(batch_job_inputs))]
```

Comment:
Changed `MockClients` to return different job_ids for batch child jobs ... in hindsight this may not have been particularly useful